src/GuppyDataExplorer/ExplorerVisualization/index.jsx

```diff
@@ -229,6 +230,7 @@ class ExplorerVisualization extends React.Component {
               className='guppy-explorer-visualization__table'
               tableConfig={{
                 fields: tableColumns,
+                ordered: tableColumnsOrdered,
```

Comment:
should update `TableConfigType` about this under https://github.com/uc-cdis/data-portal/blob/08f94f1fd08bfab0dd99055def4689eb7a6c943a/src/GuppyDataExplorer/configTypeDef.js

Reply:
i have added thanks for the heads up